Edited the scripts to append to the path in
order to locate the discovery and lib directory
under /usr/share/theharvester/

--- a/theHarvester.py
+++ b/theHarvester.py
@@ -7,6 +7,7 @@ import os
 from socket import *
 import re
 import getopt
+sys.path.append("/usr/share/theharvester/")
 
 try:
     import requests
